lib/modules/platform/github/scm.ts

```diff
@@ -0,0 +1,11 @@
+import * as git from '../../../util/git';
+import type { CommitFilesConfig } from '../../../util/git/types';
+import type { PlatformScm } from '../types';
+import { commitFiles } from './index';
+
+const githubScm: Partial<PlatformScm> = {
```

Comment:
i don't like this partial types. please create a base git scm class which forwards to util/git.
then extend that class here.